Source/Core/Core/PowerPC/Jit64/Jit.cpp

```diff
@@ -177,7 +177,7 @@ void Jit64::Init()
 
 	jo.optimizeGatherPipe = true;
 	jo.accurateSinglePrecision = true;
-	js.memcheck = SConfig::GetInstance().m_LocalCoreStartupParameter.bMMU;
+	js.memcheck = true;
```

Comment:
You actually don't need this variable at all anymore.

In the rest of the file, it's used only a few more times:

trampolines.Init(js.memcheck ? TRAMPOLINE_CODE_SIZE_MMU : TRAMPOLINE_CODE_SIZE);

farcode.Init(js.memcheck ? FARCODE_SIZE_MMU : FARCODE_SIZE);

if (js.memcheck && (opinfo->flags & FL_LOADSTORE))

Those can become

trampolines.Init(TRAMPOLINE_CODE_SIZE_MMU);

farcode.Init(FARCODE_SIZE_MMU);

if (opinfo->flags & FL_LOADSTORE)